pkg/kms/creator.go

```diff
@@ -45,21 +42,11 @@ func NewServiceCreator(c *Config) ServiceCreator {
 		keystoreID := mux.Vars(req)[keystoreIDQueryParam]
 		keyURI := fmt.Sprintf(masterKeyURI, keystoreID)
 
-		b, err := cloneRequestBody(req)
-		if err != nil {
-			return nil, err
-		}
-
-		p := struct {
-			Passphrase string `json:"passphrase"`
-		}{}
+		// TODO(#22): Replace with split secret shares. Consider to use "Hub-Kms-Secret-A" and "Hub-Kms-Secret-B-Path"
+		// headers to pass a content of secret A and a location to secret B accordingly.
+		passphrase := req.Header.Get(secretHeader)
```

Comment:
May be it makes sense to use the secret naming convention everywhere. As I understand from @Baha-sk , its secret rather than passphrase.

Reply:
updated to `secret`, but again - then we should not use the name `passphrase` in the framework code as well:
```
NewMasterLock(passphrase string, h func() hash.Hash, salt []byte)
```